x-pack/test_serverless/api_integration/test_suites/search/serverless_search/connectors.ts

```diff
@@ -19,7 +19,7 @@ export default function ({ getService }: FtrProviderContext) {
     describe('GET connectors', function () {
       before(async () => {
         supertestViewerWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
```

Comment:
nitpick: probably better to rename this variable to `supertestDeveloperWithCookieCredentials` :)